src/dmd/utils.d

```diff
@@ -132,3 +132,36 @@ extern (C++) void escapePath(OutBuffer* buf, const(char)* fname)
         fname++;
     }
 }
+
+
+/**
+ * Takes a path, and escapes '(', ')', backslashes and whitespace
+ *
+ * Params:
+ *   buf = Buffer to write the escaped path to
+ *   fname = Path to escape
+ */
+extern (C++) void escapePathSpaces(OutBuffer* buf, const(char)* fname)
```

Comment:
I moved it to the D side, but because it manipulates on paths and paths in the dmd codebase are all C strings with no facilities for conversion to/from D strings, I'm following that in here and using a C string.